src/assets/locales/de/translation.json

```
@@ -161,7 +161,7 @@
   "slotlist.save": "Slotliste speichern",
   "slotlist.upload": "Slotliste hochladen",
   "slotlist.upload.description.partOne": "Lade hier deine nicht binarisierte",
-  "slotlist.upload.description.partThree": "Die Missionsdateien findest du nach dem Speichern unter",
+  "slotlist.upload.description.partThree": "Die Eventdateien findest du nach dem Speichern unter",
```

Hier geht es um die `missions.sqm` aus dem Arma Missions(/Eden)-Editor. Das würde ich eher so belassen, da [Arma selbst von Missionen](https://community.bistudio.com/wiki/Mission.sqm) spricht
```suggestion
  "slotlist.upload.description.partThree": "Die Missionsdateien findest du nach dem Speichern unter",
```